composer.json

```diff
@@ -6,13 +6,18 @@
         "php": "^7.4"
     },
     "require-dev": {
+        "phpunit/phpunit": "^8.4|^9.0",
         "symfony/var-dumper": "5.2.x-dev"
     },
     "license": "MIT",
     "authors": [
         {
             "name": "Nahid Bin Azhar",
             "email": "nahid.dns@gmail.com"
+        },
+        {
+            "name": "Hashemi Rafsan",
```

Comment:
This section is for maintainers and core authors, not for contributors. So when you got a permit to maintain the repo, then you will be enlisted here. 
Please remove it and wait until you got a permit as an author. I think you understand :)